src/Servers/IIS/IIS/src/PublicAPI.Shipped.txt

```diff
@@ -18,35 +18,8 @@ Microsoft.AspNetCore.Http.Features.IServerVariablesFeature.this[string! variable
 Microsoft.AspNetCore.Http.Features.IServerVariablesFeature.this[string! variableName].set -> void (forwarded, contained in Microsoft.AspNetCore.Http.Features)
 Microsoft.AspNetCore.Server.IIS.BadHttpRequestException
 Microsoft.AspNetCore.Server.IIS.BadHttpRequestException.StatusCode.get -> int
-Microsoft.AspNetCore.Server.IIS.Core.IISServerAuthenticationHandler
```

Comment:
Don't touch this file, instead put a
`*REMOVED*Microsoft.AspNetCore.Server.IIS.Core.IISServerAuthenticationHandler` entry in PublicAPI.Unshipped.txt for each entry here.